src/utils/generatedImageResult.js

```diff
@@ -16,7 +16,7 @@ import p5Utils from './p5Utils';
  *
  * Accepts options as an object with property `returnTensors` so that models can pass this.config.
```

Comment:
Random but it looks like someone changed the signature of this method and did not change the JSDoc accordingly.  It no longer takes a tensor or accepts options.